### DISCUSSION TITLE
Getting started

### DISCUSSION THREAD

**Discussion by Samrj12** [kiwix/kiwix-js-windows#380] [General]
Hello there,

I was very impressed by the idea of having offline websites. Could you please help me on getting started on contributing to Kiwix? I am new to open source but do have knowledge of Java and Javascript.

**Comment by Jaifroid** [comment]
@Samrj12 You can contribute to Kiwix JS upstream at the source repository, which is here: https://github.com/kiwix/kiwix-js. To get started, please take a look at https://github.com/kiwix/kiwix-js/blob/main/CONTRIBUTING.md. Please read this all the way through, then take a look at the issues and see if there are any you could tackle. If you want to contribute to other Kiwix projects, see https://www.kiwix.org/en/support-us/code/.
